HHS/Head-Start-TTADP
# TTAHUB-498: Remove Program Type from AR Entry

## Description of change

Remove Program Type from AR Entry

## How to test

Program Type should no longer appear on the AR form. This includes all relevant views and export files.

Exporting the CSV should still populate 'Program Specialists' column and the newly added 'Program Type' column (both new line separated).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-498


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
